benchmark/storage/data_table_benchmark.cpp

```diff
@@ -85,15 +86,9 @@ static void BM_ConcurrentInsert(benchmark::State &state) {
   while (state.KeepRunning()) {
     storage::DataTable table(&block_store, layout);
     auto workload = [&](uint32_t id) {
-      // generate an undo DeltaRecord to populate on Insert
-      byte *undo_buffer = new byte[undo_size_];
-      storage::DeltaRecord *undo =
-          storage::DeltaRecord::InitializeDeltaRecord(undo_buffer, timestamp_t(0), layout, all_col_ids_);
-
+      transaction::TransactionContext txn(timestamp_t(0), timestamp_t(0), &buffer_pool);
```

Comment:
Maybe use txn_manager like with SimpleInsert, begin and commit

Reply:
Honestly we shouldn't even be using the txn_manager for SimpleInsert. That's overhead we shouldn't be measuring for a DataTable benchmark.

Reply:
Maybe change that instead then.